poc/three/load_students.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python3
+"""
+Ed-Fi API Student Data Loader
+
+This script loads student and enrollment data from a CSV file into an Ed-Fi API
+using the Data Standard 5 OpenAPI Specification.
+"""
+
+import csv
+import os
+import sys
+from typing import Dict, Optional
+import requests
+from dotenv import load_dotenv
+
+
+class EdFiAPIClient:
+    """Client for interacting with Ed-Fi API"""
+
+    def __init__(self, base_url: str, api_key: str, api_secret: str):
+        self.base_url = base_url.rstrip('/')
+        self.api_key = api_key
+        self.api_secret = api_secret
+        self.access_token = None
+        self.session = requests.Session()
+
+    def authenticate(self) -> bool:
+        """Authenticate with the Ed-Fi API and get access token"""
+        auth_url = f"{self.base_url.replace('/data/v3', '')}/oauth/token"
```

Comment:
Well, so much for using the separate Token endpoint in the environment variables! Oops. This _will_ work for the ODS/API, but not for DMS.